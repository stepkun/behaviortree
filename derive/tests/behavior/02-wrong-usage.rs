// Copyright Â© 2025 Stephan Kunz

//! Test wrong usage of behavior derive macro `Behavior` 

#[behaviortree_derive::Behavior]
struct TestBehavior;

// dummy main
fn main(){}
