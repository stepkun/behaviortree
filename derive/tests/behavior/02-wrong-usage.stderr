error: expected attribute, found derive macro `behaviortree_derive::Behavior`
 --> tests/behavior/02-wrong-usage.rs:5:3
  |
5 | #[behaviortree_derive::Behavior]
  |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not an attribute
